src/library-authoring/common/data/constants.js

```diff
@@ -16,6 +16,12 @@ export const SUBMISSION_STATUS = {
   FAILED: 'failed',
 };
 
+export const LIBRARY_ACCESS = {
+  ADMIN: 'admin',
+  STAFF: 'author',
+  USER: 'read',
+};
```

Comment:
**Note to self, no action required**:

It's weird that we make this mapping. To me, "Author" and "Read-Only" are much clearer than "Staff" and "User". I know that we're pulling this in from the V1 library editor; might be worth revisiting later.

Reply:
If we can just change the UI to say Admin/Author/Read-Only now, let's do it. I think it's a source of confusion throughout the codebase when frontend terms don't match backend terms (e.g. `chapter` vs `section`), and it's hard to change once they're established. Here we have a bit of a greenfield, so let's use that to clean up and champion consistency.

Reply:
Done.